UIKitTask/Sources/Models/DataStorageService.swift

```diff
@@ -0,0 +1,152 @@
+// DataStorageService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Тестовые данные,  заглушки. Имитация сети
+final class DataStorageService {
+    // MARK: Constants
+
+    static let shared = DataStorageService()
+
+    // MARK: Private Properties
+
+    private var currentUser: User?
+    private var stories: [Story] = []
+    private var posts: [Post] = []
+    private var recommendations: [User] = []
+    private var news: [NewNotification] = []
+
+    // MARK: Initializers
+
+    private init() {
+        fillData()
+    }
+
+    // MARK: Public Methods
+
+    func getCurrentUser() -> User {
+        currentUser ?? User(name: "none", avatarImage: "none")
+    }
+
+    func getPosts() -> [Post] {
+        posts
+    }
+
+    func getStories() -> [Story] {
+        stories
+    }
+
+    func getRecommendations() -> [User] {
+        recommendations
+    }
+
+    func getNews() -> [NewNotification] {
+        news
+    }
+
+    // MARK: Private Methods
+
+    private func fillData() {
```

Comment:
```suggestion
    private func fillData() {
```
Data пишем только там где есть тип данных Data()